### PR TITLE
[zh-cn] sync apiserver-config.v1beta1.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md

### CODE REVIEW

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 1]
```diff
@@ -52,6 +52,64 @@ EgressSelectorConfiguration 为 Egress 选择算符客户端提供版本化的
...
+
+<p>
+<!--
+TracingConfiguration provides versioned configuration for OpenTelemetry tracing clients.
+-->
+TracingConfiguration 为 OpenTelemetry 跟踪客户端提供了不同版本的配置。
+</p>
+
+<table class="table">
+<thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:76]: line 76 也需要翻译

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 2]
```diff
@@ -52,6 +52,65 @@ EgressSelectorConfiguration 为 Egress 选择算符客户端提供版本化的
...
+
+- [TracingConfiguration](#apiserver-k8s-io-v1alpha1-TracingConfiguration)
+
+- [TracingConfiguration](#apiserver-k8s-io-v1beta1-TracingConfiguration)
+
+<p>
+<!--
+TracingConfiguration provides versioned configuration for OpenTelemetry tracing clients.
+-->
+TracingConfiguration 为 OpenTelemetry 跟踪客户端提供了不同版本的配置。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:72]:
```suggestion
TracingConfiguration 为 OpenTelemetry 跟踪客户端提供版本化的配置。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 3]
```diff
@@ -52,6 +52,65 @@ EgressSelectorConfiguration 为 Egress 选择算符客户端提供版本化的
...
+</p>
+
+<table class="table">
+
+<thead><tr><th width="30%">Field</th><th><!--Description-->描述</th></tr></thead>
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:77]:
```suggestion
<thead><tr><th width="30%"><!--Field-->字段</th><th><!--Description-->描述</th></tr></thead>
```

**Pull request merged**